IntegrationTests/BraintreeAmexExpress_IntegrationTests.swift

```diff
@@ -7,7 +7,7 @@ class BraintreeAmexExpress_IntegrationTests: XCTestCase {
     
     func testGetRewardsBalance_returnsResult() async {
         let apiClient = BTAPIClient(authorization: BTIntegrationTestsConstants.sandboxClientTokenVersion3)!
-        let cardClient = BTCardClient(apiClient: apiClient)
+        let cardClient = BTCardClient(authorization: "sandbox_9dbg82cq_dcpspy2brwdjr3qn")
```

Comment:
Should this be using the same authorization? Looks like it was `BTIntegrationTestsConstants.sandboxClientTokenVersion3`.